### ISSUE TITLE
Emacs hangs after typing M-x plsense-server-start

### ISSUE THREAD

**Issue by kb01** [aki2o/emacs-plsense#7]
Hello,

After typing M-x plsense-server-start, Emacs doesn't respond anymore: I tried to reset my iptables configuration but the same problem occurs always. This is the error message:

[PLSENSE] Start server ...
[PLSENSE] Start server is failed.
command-execute: Buffer is read-only: #<buffer *Messages*>

How to solve this issue?

Thanks in advance.

**Comment by aki2o** [comment]
Hi, @kb01 
Try to do `plsense svstart` in commandline. What will happen?

**Comment by kb01** [comment]
No output message gets generated and the program doesn't stop running on the shell, after typing this command on the shell (under normal user privileges, not root).

**Comment by kb01** [comment]
@aki2o I'm getting successive similar 5 error messages: "Argument "__Positional_0001" isn't numeric in postincrement (++) at /usr/share/perl5/site_perl/Class/Std/Storable.pm line 51." after typing plsense svstart.

**Comment by aki2o** [comment]
Mmm... Sorry... I don't seem to be able to resolve. It's no happen in my environment.
Does `plsense svstart` after deleting `cachedir` cause the same error too ?

And...

1. What version of Perl are you using?

2. The installed `Class::Std::Storable` version might be different from mine because it seems to have the cause.
Here is the version and md5sum value in my environment.
```shell
$ perl -e "use Class::Std::Storable 9999"
Class::Std::Storable version 9999 required--this is only version 0.0.1 at -e line 1.
BEGIN failed--compilation aborted at -e line 1.
$ perldoc -l Class::Std::Storable
/home/hiroaki/perl5/perlbrew/perls/perl-5.18.4/lib/site_perl/5.18.4/Class/Std/Storable.pm
$ md5sum /home/hiroaki/perl5/perlbrew/perls/perl-5.18.4/lib/site_perl/5.18.4/Class/Std/Storable.pm
a20f34d2dd8491089cb0301d942a64e8  /home/hiroaki/perl5/perlbrew/perls/perl-5.18.4/lib/site_perl/5.18.4/Class/Std/Storable.pm
```

**Comment by kb01** [comment]
Well, the problem still occurs after removing ~/.plsense.d and I got:

[PLSENSE] Start server ...
[PLSENSE] Start server is failed.
byte-code: End of buffer [9 times]
byte-code: End of buffer

along, with the same 5 identical error messages in the terminal, as described above.

Additional infos:

1- 5.22.0
2- $ perl -e "use Class::Std::Storable 9999"
Class::Std::Storable version 9999 required--this is only version 0.0.1 at -e line 1.
BEGIN failed--compilation aborted at -e line 1.
$ perldoc -l Class::Std::Storable
/usr/share/perl5/site_perl/Class/Std/Storable.pm

So what is supposed to be wrong here?

note: the 5 identical error messages, as described above, in the CLI, got printed just after I type plsense-server-start, I'm using Emacs v24.4: I just attached my dotemacs configuration file here.
